data-prepper-api/src/main/java/org/opensearch/dataprepper/model/event/JacksonEvent.java

```diff
@@ -91,18 +91,21 @@ protected JacksonEvent(final Builder builder) {
         }
 
         this.jsonNode = getInitialJsonNode(builder.data);
+        this.eventHandle = new DefaultEventHandle(eventMetadata.getTimeReceived());
     }
 
     protected JacksonEvent(final JacksonEvent otherEvent) {
         this.jsonNode = otherEvent.jsonNode.deepCopy();
         this.eventMetadata = DefaultEventMetadata.fromEventMetadata(otherEvent.eventMetadata);
+        this.eventHandle = new DefaultEventHandle(eventMetadata.getTimeReceived());
     }
 
     public static Event fromMessage(String message) {
-        return JacksonEvent.builder()
+        JacksonEvent event = JacksonEvent.builder()
```

Comment:
What does this change accomplish?

Reply:
Not needed. I was trying something. undid the change. Thanks!